code-studio/src/js/initApp/project.js

```diff
@@ -344,7 +344,7 @@ var projects = module.exports = {
   getStandaloneApp: function () {
     switch (appOptions.app) {
       case 'applab':
-        return 'applab';
+        return appOptions.level.makerlabEnabled ? 'makerlab' : 'applab';
```

Comment:
This feels safe to access in this context as it hits the same code `appOptions.level.*` path as [this line](https://github.com/code-dot-org/code-dot-org/pull/8009/files#diff-25c0ec2f1d9f27e57d4d8b1f8fdb0199R357)